### DISCUSSION TITLE
Bug immutable package

### DISCUSSION THREAD

**Discussion by NmurtasDev** [Netcentric/accesscontroltool#601] [Q&A]
Hi, the project I'm working on required the installation of ACTool but during the integration procedures (suggested to me by the adobe forum "https://experienceleaguecommunities.adobe.com/t5/adobe-experience-manager/deploying-netcentric-access-control-tool-with-aem-project/td-p/395463") I found this error 

`com.myproject:aem-myproject-project.all:0.0.1-SNAPSHOT,0,Inclusion of both /etc/packages/Netcentric/accesscontroltool-package-3.0.2.zip/jcr:content and /apps/myproject-packages in the same package violates the separation of immutable and mutable paths in the same package.,Code Smell,Minor,,ImmutableMutableMixedPackage,"aem,cloud-service-compatibility",https://www.adobe.com/go/aem_cmcq_immutablemutablemixe_en`

The devops report me the same error when try to install the project inside the Adobe Cloud.
What is the right way to supplement Actool?

PS reading the guide I found this reference to adobe cloud 
reading it I didn't understand if it refers to the tool installation or to the permissions installation 
https://github.com/Netcentric/accesscontroltool/blob/develop/docs/ApplyConfig.md#installation-in-aem-as-a-cloud-service
In case the guide is about installation, do we need to coordinate with the Adobe team to proceed with the installation?

**Comment by kwin** [comment]
Please make sure that all your sub packages in aem-myproject-project.all end up in the same folder (either below /apps/myproject-packages) or below (/etc/packages/), but do not mix both approaches.

**Reply by adellanegra-zz** [reply]
Hi @kwin ,
I'm a colleague of @NmurtasAlp , we applied the configuration in order to include the accesstool packages below /apps/myproject-packages but, the two packages themeselves have their own pom files with filters applied into /etc/packages/Netcentric, so even if we include them below /apps, the installation process put them below /etc/packages/Netcentric and creates the new folder /apps/netcentric:
![image](https://user-images.githubusercontent.com/10206887/137305471-cf3513d4-127b-4e95-959e-988bd26fb2c5.png)
![image](https://user-images.githubusercontent.com/10206887/137305546-6a6b52aa-6c5f-411a-a56b-f380f2b04ecb.png)
![image](https://user-images.githubusercontent.com/10206887/137305630-4a64f9a1-3e33-48ad-94eb-ba83b7d646e6.png)

**Reply by kwin** [reply]
Is the CodeSmell emitted by CloudManager for AMS or AEMaaCS? I have not yet seen this so the rule must have been modified recently. Can you share how the container package (http://jackrabbit.apache.org/filevault/packagetypes.html) looks exactly in your case (the one embedding actool)?
